Middlewares/ST/STM32_USB_Device_Library/Class/MSC/Src/usbd_msc_scsi.c

```diff
@@ -502,6 +508,12 @@ static int8_t SCSI_ModeSense10(USBD_HandleTypeDef *pdev, uint8_t lun, uint8_t *p
     return -1;
   }
 
+  /* Check If media is write-protected */
+  if (((USBD_StorageTypeDef *)pdev->pUserData)->IsWriteProtected(lun) == 1)
```

Comment:
`pUserData` should become `pUserData[pdev->classId]`.

Reply:
Thanks! Brunch merged with actual master and fixit with commit:
[59562f2](https://github.com/STMicroelectronics/STM32CubeL4/pull/42/commits/59562f2d21e012c200830446e75d4418e56533b1)